ConwaysGameOfLife/OutputFormatter.cs

```diff
@@ -1,32 +1,32 @@
 using System;
+using System.Text;
 
 namespace ConwaysGameOfLife
 {
-    public class OutputFormatter 
+    public static class OutputFormatter 
     {
-        // displays universe & instructions
-        // TODO: try StringBuilder 
+        // formats universe display 
 
-        public string FormatUniverse(Universe universe) // universe created with user input - dimensions & live cells ?? - needed??
+        public static string FormatUniverse(Universe universe) // universe created with user input - dimensions & live cells ?? - needed??
         {
-            var format = "";
+            StringBuilder format = new StringBuilder();

```

Comment:
Does the `format` name have any significance? Perhaps we could call it `sb` or `stringBuilder` or `universeBuilder` (examples only - make/pick one that makes sense to you). Also, we can still use the implicit `var` type instead of explicit `StringBuilder`.

```suggestion
            var sb = new StringBuilder();
```